yt_dlp/extractor/nick.py

```diff
@@ -75,10 +75,6 @@ def _get_feed_query(self, uri):
             'mgid': uri,
         }
 
-    def _extract_mgid(self, webpage):
```

Comment:
u just removed stuff from here and it worked?

Reply:
Since I added the removed code to the main infoextractor that it uses there wasn't a need to have it in both the mtv infoextractor and this one. It's kind of redundant so I removed it. And yes it works. :-)